### PR TITLE
Removal of PubNub SDK v6 & Update Codebase to v7

### PR DESCRIPTION
**Pull request by olivercarson-pubnub**
-Removal of PubNub SDK v6 (Old SDK)
-Adding new PubNub SDK (v7)
-Update codebase to use new SDK (v7)

Note: There are still some issues with Friends List acting odd (status is not showing, adding self to the friend list for some reason). This whole system will be reworked in [DEVX-3359](https://pubnub.atlassian.net/browse/DEVX-3359).

### CODE REVIEW

**Comment by olivercarson-pubnub** [comment]
The PR is ready for review. The main focus is on Assets/SuperMultiplayerShooter/Scripts, especially the removal of the old Unity SDK and updates to the Multiplayer Sync. The new Unity SDK package is included in these commits.

File: Assets/SuperMultiplayerShooter/Game.unity [hunk 1]
```diff
@@ -104,7 +104,7 @@ NavMeshSettings:
   serializedVersion: 2
```
Comment (darryncampbell-pubnub) [inline Assets/SuperMultiplayerShooter/Game.unity:104]: Presumably, the changes to Game.Unity were required to upgrade the PubNub SDK?

File: Assets/SuperMultiplayerShooter/Graphics/CosmeticCharacterPreview.renderTexture [hunk 2]
```diff
@@ -12,12 +12,13 @@ RenderTexture:
     Hash: 00000000000000000000000000000000
```
Comment (darryncampbell-pubnub) [inline Assets/SuperMultiplayerShooter/Graphics/CosmeticCharacterPreview.renderTexture:12]: Again, was this file changed deliberately?

**Comment by darryncampbell-pubnub** [comment]
Hi @olivercarson-pubnub I started going through this but I stopped as I had a couple of big concerns:

- You have upgraded the version of Photon which I believe has updated a lot of other files such as the prefabs (as one example).  Basically, there are a LOT of file changes that weren't related to the Unity SDK.  Another example are all the *.unity files
- The changes I made to main for the multi-player sync when we merged my pull request... you haven't merged those changes back into your branch so you have lots of merge conflicts.  At least I think that's what is happening, it seems strange that I'm seeing MY changes as part of your pull request?

**Pull request closed without merging**